assignment1.js

```diff
@@ -0,0 +1,80 @@
+/*Palindrome Permutation: Given a string, write a function to check if it is a permutation of a palin-
+drome. A palindrome is a word or phrase that is the same forwards and backwards. A permutation
+is a rearrangement of letters. The palindrome does not need to be limited to just dictionary words.
+EXAMPLE
+Input: T a c t Coa
+Output: T r u e ( p e r m u t a t i o n s : " t a c o c a t " , " a t c o e t a " , e t c . )*/
+
+function isPalindrome(query) {
+    let lowQ = query.toLowerCase().split(" ").join("");
+    let letters = {};
+    let count = 0;
+
+    for(let char of lowQ) {
+        if (!letters[char]) {
+            letters[char] = 1;
+        }
+        else {
+            letters[char]++;
+        }
+    }
+    for(l in letters) {
+        if(letters[l] % 2) {
+            count++;
+        }
+        if(count > 1) {
+            return false;
+        }
+    }
```

Comment:
Although there's nothing wrong with how you've written this (other than the cramped spacing), there is a cleaner way to do this using JS's `Object.values` method. Because that method returns an array, you can then use `Array.prototype.filter` to keep only the odd values, and then compare the length of that array against `1` like you're already doing.
```suggestion
    return Object.values(letters).filter(value => value % 2 !== 0).length <= 1;
```